packages/grafana-ui/src/components/ValueMappingsEditor/ValueMappingsEditor.tsx

```diff
@@ -68,13 +68,21 @@ export const ValueMappingsEditor = React.memo(({ value, onChange }: Props) => {
         {rows.length > 0 && <span>Edit value mappings</span>}
         {rows.length === 0 && <span>Add value mappings</span>}
       </Button>
-      <Modal isOpen={isEditorOpen} title="Value mappings" onDismiss={onCloseEditor} className={styles.modal}>
+      <Modal
+        isOpen={isEditorOpen}
+        title="Value mappings"
+        onDismiss={onCloseEditor}
+        className={styles.modal}
+        onClickBackdrop={DoNothingBackdropClickFunction}
```

Comment:
Can you maybe change the API of Modal to have `closeOnBackdropClick` instead of this noop?

Reply:
yea, that is probably better